code/processes/gateway.q

```diff
@@ -458,8 +381,8 @@ asyncexecjpts:{[query;servertype;joinfunction;postback;timeout;sync]
    if[99h<>type servertype;
      // its a list of servertypes e.g. `rdb`hdb
      servertype:distinct servertype,();
-     missing:servertype except exec distinct servertype from .gw.servers where active;
-     if[count missing; errStr:.gw.errorprefix,"not all of the requested server types are available; missing "," " sv string missing];
+     errcheck:@[getserverids;servertype;{.gw.errorprefix,x}];
+     if[10h=type errcheck; errStr:errcheck];
```

Comment:
don't mix in camel case here (errStr)... I know that's the way it is in the old code but better to remove that now